### PR TITLE
Added beginners course for python

### PR DESCRIPTION
**Pull request by daxeel**

### CODE REVIEW

**Comment by rathodchintan** [comment]
Can you please check format we have used in https://github.com/GDGAhmedabad/Learning-Resources/blob/master/android/blogs.md and https://github.com/GDGAhmedabad/Learning-Resources/blob/master/android/courses.md files. Please format files and add PR again.

**Comment by daxeel** [comment]
ok

**Comment by rathodchintan** [comment]
Perfect.. :)

File: Python/books.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Name | Links 
+------------ | ------------- 
+Introduction to computer science using Python - One of the best books for beginners. | http://it-ebooks.info/book/2466/
```
Comment (PareshMayani) [inline Python/books.md]: please include an original URL of the book! If our member wants to read that particular book then he/she will try to search and get e-book him/her self!

Comment (daxeel) [inline Python/books.md]: I didn't get you by "Original url". I have provided url from anyone can download ebook as pdf.

Comment (PareshMayani) [inline Python/books.md]: For example: https://www.packtpub.com/application-development/android-programming-beginners

**Comment by PareshMayani** [comment]
Sorry for the confusion about book links and formatting issue! Closing this PR and looking forward for your next PR with proper formatting and links. Thanks for the contribution, really appreciate it.

**Pull request closed without merging**